drivers/Makefile.include

```diff
@@ -133,3 +133,7 @@ endif
 ifneq (,$(filter sx127%,$(USEMODULE)))
     USEMODULE_INCLUDES += $(RIOTBASE)/drivers/sx127x/include
 endif
+ifneq (,$(filter winc1500,$(USEMODULE)))
+    USEMODULE_INCLUDES += $(RIOTBASE)/drivers/winc1500/include
```

Comment:
The 4 spaces indention is used eveywhere in this file though. I am keeping it.

Reply:
Indeed, but this is because your branch is not up-to-date with latest master. The 2 spaces indent is the default for Makefile and have recently been changed in this file by #7576

Reply:
Oh I see. I will make it 2 spaces then.